sim_3d_volumes.py

```diff
@@ -0,0 +1,103 @@
+""" Generate 3D map of molecules
+"""
+import os
+import numpy as np
+import torch
+import coords
+
+
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    Main_dir = "/scratch/ex-kdd-1/NicolasLegendre/Cryo/Data"
+else:
+    Main_dir = os.getcwd() + "\\Data\\"
+
+Particules = np.asarray([[1, 0, 0, 3], [0, 8, 0, 5], [0, 0, 7, 9]])
+N_particules = 2000
+Vol_size = 64
+Name = '4_points_3D'
+Long = 2
```

Comment:
I would not hard code those parameters here. Please see comment below and make these arguments default of `main()`